syntax = "proto3";

message MealyMachine { string machine = 1; }

message MealyTransitions { string transitions = 1; }